keps/sig-api-machinery/3352-aggregated-discovery/README.md

```diff
@@ -766,6 +795,12 @@ the previous answers based on experience in the field. -->
 Kubernetes API (e.g., checking if there are objects with field X set)
 may be a last resort. Avoid logs or events for this purpose. -->
 
+Operators can check whether an aggregated discovery request can be
+made by sending a request to `apis` with
+`application/json;as=APIGroupDiscoveryList;v=v2beta1;g=apidiscovery.k8s.io,application/json`
+as the Accept header and looking at the return type. They can also
```

Comment:
be more specific than "return type"?

Reply:
Updated to Content-Type and provided the actual return values to check for.